### DISCUSSION TITLE
请问写package的时候，怎么指定target？

### DISCUSSION THREAD

**Discussion by vkensou** [xmake-io/xmake#4611] [Q&A] [answered]
如果一个项目里有多个target，然后我想把这个项目做成一个package，那在写package的xmake.lua时，怎么指定具体使用哪一个target呢？

我看了几个repo里的package，没找到有做这一步的地方。比如tbox项目里有多个target：tbox和demo，但我没看见tbox package的xmake.lua里有指定使用哪个target？

**Comment by star-hengxing** [comment]


**Reply by waruqi** [reply, marked as answer]
```
import("package.tools.xmake").install(package, configs, {target = "xxxx"})
```

**Reply by vkensou** [reply]
非常感谢解答。如果没有指定target，那最终用的是哪一个？最后一个吗？

**Reply by waruqi** [reply]
所有
